lib/mbtiles.js

```diff
@@ -52,11 +52,22 @@ function MBTiles(uri, callback) {
     uri.query = uri.query || {};
     if (!uri.query.batch) uri.query.batch = 100;
 
+    if (!uri.query.mode) uri.query.mode = 'rwc';
+    var flagEnum = {
+        ro: sqlite3.OPEN_READONLY,
+        rw: sqlite3.OPEN_READWRITE,
+        rwc: sqlite3.OPEN_READWRITE | sqlite3.OPEN_CREATE
```

Comment:
@jingsam can you add a comment here just to provide context of the bitwise so we have some documentation around it? My understanding here is that we're converting the two enums into a mode that allows `OPEN_READWRITE` & `OPEN_CREATE`, rather than a mode that is _just_ `OPEN_CREATE`?

Reply:
https://www.sqlite.org/c3ref/open.html

```
 If "ro" is specified, then the database is opened for read-only access, just as if the SQLITE_OPEN_READONLY flag had been set in the third argument to sqlite3_open_v2(). 

If the mode option is set to "rw", then the database is opened for read-write (but not create) access, as if SQLITE_OPEN_READWRITE (but not SQLITE_OPEN_CREATE) had been set. 

Value "rwc" is equivalent to setting both SQLITE_OPEN_READWRITE and SQLITE_OPEN_CREATE.
```